monero-outreach-docs/translations/fr/storyboard/randomX_fr.md

```diff
@@ -0,0 +1,36 @@
+
+RandomX a deux modes avec des besoins de mémoire et des performances différents. Le mode rapide nécessite 2 Go de mémoire partagée, mais 4x-6x les performances du mode lumière qui ne nécessite que 256 Mo de RAM. Le mode rapide est destiné aux mineurs dédiés. Le mode Light est conçu pour permettre aux nœuds complets de valider des blocs sans avoir besoin de 2+Go de RAM, de sorte que les petits périphériques (comme les ordinateurs monocartes ARM, par exemple Rock64) peuvent toujours être utilisés comme des nœuds autonomes. 
+
+| CPU | OS | Threads | RAM | CryptoNight-R (v8) | RandomX Fast Mode | RandomX Light Mode |
+|--|--|--|--|--|--|--|
+| AMD Ryzen 7 1700 | Ubuntu 16.04 | 8 | 16 GB DDR4 | 650 H/s | 4100 H/s | 620 H/s |
+| Intel Core i7-8550U | Windows 10 | 4 | 16 GB DDR4 | 240 H/s | 1700 H/s | 350 H/s |
+| Intel Core i3-3220 | Ubuntu 16.04 | 4 | 4 GB DDR3 | 75 H/s | 510 H/s | 150 H/s |  
+
+| GPU | Clockspeed | CryptoNight-R (v8) | RandomX |
+|--|--|--|--|
+| GTX 1660 Ti | 2070/13760 MHz | 626 H/s (98 W) | 660 H/s (103 W) |
+| GTX 1080 Ti | 1930/10010 MHz | 787 H/s (145 W) | 1136 H/s (190 W) |  
+
+## _La Collaboration RandomX_ 
+
+RandomX a été développé pour Monero par tevador, hyc, vielmetti, antanst et SChernykh. D'autres organisations sont déjà intéressées par son adoption. [Arweave](https://www.arweave.org/), un facilitateur de stockage sans serveur a fait don de l'audit Trail of Bits à la communauté Monero et va implémenter RandomX avant Monero pour leur propre utilisation. Arweave fournit une approche innovante basée sur les cryptomonnaies pour le stockage décentralisé à long terme des données. L'exploitation minière sur Arweave repose à la fois sur la preuve du travail et - comme nouveau concept unique à Arweave - sur la preuve d'accès. Les mineurs d'Arweave sont incités par la preuve d'accès à la duplication et ont un accès rapide aux données stockées dans le réseau. 
+
+[Wownero](http://wownero.org/) lance également RandomX dans sa prochaine mise à jour v0.6 et l'appellera RandomWOW. Le code RandomX sera mis à jour une fois les audits terminés, de façon a être prêt pour le prochain "fork" du réseau Monero. Une autre différence est que RandomWOW aura un scratchpad plus petit pour le hachage, 1Mo au lieu de 2Mo, un plus petit nombre d'itérations d'exécution de VM, et une augmentation des exécutions de VM en chaîne par hachage pour augmenter le coût de compilation du programme pour les GPU. 
```

Comment:
`de façon a être prêt pour le prochain "fork" du réseau Monero` => `il y aura donc quelques divergence du code lors de la prochaine mise à jour du réseau Monero`  
`Une autre différence est que RandomWOW aura un scratchpad plus petit pour le hachage` => `Une autre différence réside dans le presse-papier de hachage plus petit de RandomWOW`  
`un plus petit nombre d'itérations d'exécution de VM` => `le plus petit nombre d'itérations d'exécution de VMs`  
`et une augmentation des exécutions de VM en chaîne par hachage` => `et l'augmentation du chaînage des exécutions de VMs`  
`pour augmenter` => `afin d'augmenter`

Reply:
quelques divergence -> divergences